src/style.css

```diff
@@ -0,0 +1,49 @@
+* {
+  margin: 0;
```

Comment:
Don't reset margins and paddings in the root selector. Do this in `body` selector